traceapp/tmpl/data/trace.html

```diff
@@ -420,6 +428,94 @@
     if(data != null) {
       timelineHover();
     }
+
+    // save state for last sort
+    var lastSort = {
+      name: '',
+      order: false,
```

Comment:
Instead of naming this just `order`, could we rename it to `ascendingOrder` or `descendingOrder` (depending on which state `true` represents here)?